cibuildwheel/architecture.py

```diff
@@ -77,6 +81,18 @@ def all_archs(platform: PlatformName) -> 'Set[Architecture]':
         else:
             assert_never(platform)
 
+    @staticmethod
+    def bitiness_archs(platform: PlatformName, bitiness: Literal['64', '32']) -> 'Set[Architecture]':
```

Comment:
Oops I thought I put a comment about this earlier, must have deleted it accidentally. I think the word is 'bitness', 'bitiness' :)